### DISCUSSION TITLE
hasn't support FPV controller mapping

### DISCUSSION THREAD

**Discussion by raining1123** [LizardByte/community#518] [Sunshine]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Is your issue described in the documentation?

- [X] I have read the documentation

### Is your issue present in the latest beta/pre-release?

This issue is present in the latest pre-release

### Describe the Bug

I want stream some FPV simulator games, like Uncrushed, Liftoff, etc.  
but it does not work when I plug my FPV controller (DJI FPV 3) in the client side (laptop nor android tablet).
it seems no mapping for now while XBOX controller is working well.

### Host Operating System

Windows

### Operating System Version

Windows 10

### Architecture

64 bit

### Sunshine commit or version

v2024.928.34947

### Package

Windows - installer

### GPU Type

Nvidia

### GPU Model

4060Ti 16GB

### GPU Driver/Mesa Version

560.94

### Capture Method

Windows.Graphics.Capture (Windows)

### Config

```Shell
channels=5
address family = both
sunshine_name=bomber
locale = zh
upnp =enabled
origin_web_ui_allowed = wan 
wan_encryption_mode=0
notify_pre_releases = enabled
```

### Relevant log output

```shell
no relevant with log , it is a mapping issue
```

**Comment by ReenigneArcher** [comment]
Seems unlikely since we rely on vigembug for gamepad input on Windows, but I've converted this to a feature request.

**Comment by t4chik0ma** [comment]
+1 for this, should be able to support using controllers other than XB/PS
